IssuesData.PullRequestId could not be found

IssuesData.PullRequestId could not be found:

```
C:/Git/Azure/Cake.Issues.Recipe-Demo/tools/Cake.Issues.Recipe.0.2.1/Content/tasks/pullrequestsystems/AzureDevOpsPullRequestSystem.cake(29,22): error CS1061: 'IssuesData' does not contain a definition for 'PullRequestId' and no accessible extension method 'PullRequestId' accepting a first argument of type 'IssuesData' could be found (are you missing a using directive or an assembly reference?)
```

--- a/Cake.Issues.Recipe/Content/tasks/pullrequestsystems/AzureDevOpsPullRequestSystem.cake
+++ b/Cake.Issues.Recipe/Content/tasks/pullrequestsystems/AzureDevOpsPullRequestSystem.cake
@@ -26,7 +26,7 @@ public class AzureDevOpsPullRequestSystem : BasePullRequestSystem
             data.Issues,
             context.TfsPullRequests(
                 data.BuildServer.DetermineRepositoryRemoteUrl(context, data.RepositoryRootDirectory),
-                data.PullRequestId.Value,
+                data.BuildServer.DeterminePullRequestId(context).Value,
                 context.TfsAuthenticationOAuth(context.EnvironmentVariable("SYSTEM_ACCESSTOKEN"))),
             data.RepositoryRootDirectory);
     }
@@ -53,7 +53,7 @@ public class AzureDevOpsPullRequestSystem : BasePullRequestSystem
         var pullRequestSettings =
             new TfsPullRequestSettings(
                 data.BuildServer.DetermineRepositoryRemoteUrl(context, data.RepositoryRootDirectory),
-                data.PullRequestId.Value,
+                data.BuildServer.DeterminePullRequestId(context).Value,
                 context.TfsAuthenticationOAuth(context.EnvironmentVariable("SYSTEM_ACCESSTOKEN")));
 
         var pullRequstStatus =

